packages/page-assets/src/Balances/index.tsx

```diff
@@ -39,9 +40,9 @@ function Balances ({ className, infos = [] }: Props): React.ReactElement<Props>
   );
 
   const assetOptions = useMemo(
-    () => completeInfos.map(({ id, metadata }, index) => ({
+    () => completeInfos.map(({ id, metadata }) => ({
       text: `${metadata.name.toUtf8()} (${formatNumber(id)})`,
-      value: index
+      value: id.toNumber()
```

Comment:
Why do we replace `index` here?

Reply:
Because if use index then we can't exact match by asset ID.

Reply:
Makes total sense. thanks!